README.md

```diff
@@ -93,22 +93,28 @@ $ make_prg from_msa --help
 usage: make_prg from_msa [options] <MSA input file>
 
 positional arguments:
-  MSA                   Input file: a multiple sequence alignment in supported alignment_format. If not in aligned fasta
-                        alignment_format, use -f to input the alignment_format type
+  MSA                   Input file: a multiple sequence alignment in supported alignment_format. If not in aligned fasta alignment_format, use -f to input
```

Comment:
What does `If not in aligned fasta alignment_format` mean? I think should be `If not in aligned fasta format`?

Reply:
Urgh, yeah that was worded weird. I have made it much more succinct